### DISCUSSION TITLE
In Qfield it would be an amazingly useful to have some basic annotation drawing tools for sketching over a base map

### DISCUSSION THREAD

**Discussion by greengadget** [opengisch/QField#5011] [Ideas]
My company does outdoor field surveys which have to be quick and efficient. Making a transition from paper based recording has had many benefits (photos tagged to a point layer etc) but there remains a basic need for rapid drawing and note taking to highlight features and scribble notes on a map. QGIS now has a layout export tool that allow creation of geolocated pdfs but I cannot find any app or feature within Qfield that will allow me to see my current position on a basemap and allow me to quickly annotate over the top with my tablet stylus. My colleagues are crying out for a way to do this and so far I cannot provide them with a way to do it.
Yes we can now use the free form tool when editing a line layer and this is useful up to a point but it not replicating the ease of use of a pencil or pen. I dont want to have to click a tick to accept the creation of every word I scribble down! Would it be possible to create a pdf annotation tool, or a basic drawing tool that could edit a raster layer as an overlay? Any suggestions very welcome. Thanks!

**Comment by VxTedxV** [comment]
+1 about quick notes
Workaround : set a point layer where you can create point where you are located and set a text field that will be the label of each point

**Comment by myxBeni** [comment]
Definitely agree. Some way to scribble a note would be very useful. Is there a PDF reader for Android that features real-time geolocation? 
I didn't try the [eraser-tool](https://docs.qfield.org/how-to/digitize/#reshape-eraser-tool) so far, but this should allow easier line-sketches. If you don't add any attributes to the lines, you don't need to confirm them.
Having different layers would be possible, but paper is so versatile, because you can switch from lines to polygons to annotations.
For QGIS, [RedLayer](https://plugins.qgis.org/plugins/redLayer/#plugin-details) was supposed to to achieve that... but on desktops.
